### PR TITLE
Add test suite for utility.py

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds a test suite for utility.py file.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by pkubatrh** [comment]
Otherwise lgtm

File: tests/test_utility.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        self.ir = ImageRebuilder('Testing')
+
+        self.ir.set_config('default.yaml', release="rawhide")
+
+    @pytest.mark.parametrize(
+        "config,expected_path,expected_image_set",
+        [
+            ("fedora35.yaml", "fedora35.yaml", "current"),
+            ("fedora35.yaml:f35", "fedora35.yaml", "f35"),
+            ("fedora35.yaml:f35:something", "fedora35.yaml", "f35"),
```
[the remark this answers is not in the document]
Comment (phracek) [reply tests/test_utility.py]: This should be fixed by commit https://github.com/sclorg/container-workflow-tool/pull/60/commits/6f9eebdb6323d85a458baa563de85a086db04492

**Comment by phracek** [comment]
@pkubatrh Please have a look at it one more time.

File: container_workflow_tool/utility.py [hunk 2]
```diff
@@ -51,23 +51,30 @@ def _remove_file(path, logger=None):
...
+
 def _split_config_path(config: str) -> (str, str):
-        conf = config.split(':')
-        config_path = conf[0]
-        image_set = conf[1] if len(conf) > 1 else 'current'
-        return (config_path, image_set)
+    conf = config.split(':')
+    if len(conf) > 2:
+        raise RebuilderError(
+            "Config path contains a lot of arguments."
```
Comment (pkubatrh) [inline container_workflow_tool/utility.py:71]:
```suggestion
            "You can only use one image_set with the --config option."
```

File: container_workflow_tool/utility.py [hunk 3]
```diff
@@ -51,23 +51,30 @@ def _remove_file(path, logger=None):
...
+            "It has to be only <config.yaml>:<image_set>."
```
Comment (pkubatrh) [inline container_workflow_tool/utility.py:72]: I would remove this line - the help string already has an example.

Comment (phracek) [reply container_workflow_tool/utility.py]: Resolved by https://github.com/sclorg/container-workflow-tool/pull/60/commits/6870de693fe7a6c1112d8fea0b7565e8bdcfeb27

File: container_workflow_tool/utility.py (hunk 2 above)
Comment (pkubatrh) [reply container_workflow_tool/utility.py]: I would still like to use this suggestion, as this error message is more specific.

Comment (phracek) [reply container_workflow_tool/utility.py]: IT was marked by me as Resolved and I did not get it. Sorry, I will fix it.

**Comment by pkubatrh** [comment]
lgtm now

**Pull request merged**